src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] [OPTIONAL] Your app produces a warning, kindly fix it so your app runs without warnings/errors, also try to not ignore it by adding `/ eslint-disable-next-line`

Reply:
/ eslint-disable-next-line has been removed from the js file. Also, the bug is fixed.